crt_portal/cts_forms/templates/forms/report_class.html

```diff
@@ -1,39 +1,41 @@
 {% extends "forms/report_base.html" %}
 {% load static %}
 {% block form_questions %}
+
 <fieldset class="usa-fieldset" aria-label="{{ current_step_name }}">
-  <legend class="usa-sr-only">{{ current_step_name }}</legend hidden>
```

Comment:
Mind if we add the legend back?  I think we could remove the aria label. That way, I think it will be accessible and tests will pass.

Reply:
Oooooh, great catch! Thanks for spotting this @LindsayYoung! 🏆 

A couple of notes here on the `<legend>` implementation that I removed, for learning purposes: 

1. The `hidden` attribute is on the closing tag, should be on the opening tag.
2. The `hidden` attribute and `usa-sr-only` class shouldn't be on the same element. The `hidden` attribute will hide an element for both visual users and screen reader users, so that conflicts with the intent of `usa-sr-only`.

If the element were really `hidden`, my expectation is that pa11y would return the same failure as if the element were not there at all. So maybe this implementation was working because the `hidden` attribute on the closing tag was ignored.

Will investigate further this afternoon after a few meetings wrap up!

Reply:
In this case, the text that's currently marked up as the label for protected class describes the different checkbox input buttons:

> "Do you believe you were treated this way because of any of the following characteristics or statuses that apply to you?"

Semantically, that feels equivalent to a `<legend>`. I switched that text over to `<legend>` which addresses the pa11y issue, has no style impact, and feels like the right thing semantically. For more context: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset